libbeat/testing/integration/sample_test.go

```diff
@@ -0,0 +1,93 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package integration
+
+import (
+	"context"
+	"testing"
+	"time"
+)
+
+func TestFilebeat(t *testing.T) {
```

Comment:
It's not a big deal, but it feels a bit odd to have libbeat testing Filebeat. We use mockbeat (`libbeat/mock/mockbeat.go`) to test many parts of libbeat, including with system tests.

I believe it makes more sense to either test mockbeat here or to move this to Filebeat's test folder.

Reply:
You're right. This test was meant to be a demo test to show how to use the framework.  I'm going to delete it from here since Filebeat itself already has a sample test that covers even more. We can use the entire `filebeat/testing/integration` package as a demo to this framework.